app/lib/app_env.rb

```diff
@@ -0,0 +1,6 @@
+# A class which overrides the default indexing behavior of ENV.
```

Comment:
I'm not sure the comment adds much value here due to the simplicity of the code.

Reply:
Removed the comment - I completely agree that it is an unnecessary annotation of the class.